src/MANYBODY/pair_rebo.cpp

```diff
@@ -30,9 +30,263 @@ void PairREBO::settings(int narg, char **arg)
 
   cutlj = 0.0;
   ljflag = torflag = 0;
+ PCCf_2_0 = -0.0276030;
```

Comment:
Why is this still here? Can't the `PCCf_2_0` member just be removed, with the value inlined back into `pair_airebo.cpp`?